wicket-util/src/main/java/org/apache/wicket/util/crypt/ClassCryptFactory.java

```diff
@@ -28,6 +28,8 @@
  * must implement {@link ICrypt}.
  * 
  * @author Igor Vaynberg (ivaynberg)
+ * 
+ * @deprecated use a lambda expression instead TODO remove in Wicket 10
  */
 public class ClassCryptFactory implements ICryptFactory
```

Comment:
Do we still need `ICryptFactory` ? Or it should be deprecated too ?

Reply:
CryptoMapper still utilizes SecuritySettings#cryptFactory -> ICryptFactorty